frontend/src/app/model/types.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { AGGREGATION_FUNCTION_SETTINGS, ALL_AGGREGATION_FUNCTIONS } from './types';
+
+describe('types', () => {
```

Comment:
is that needed ;)?

Reply:
Indeed I can remove that for now. I added it in preparation of opcua scan groups implementation later.